kaldb/src/main/java/com/slack/kaldb/server/KaldbIndexer.java

```diff
@@ -151,17 +151,13 @@ protected void run() throws Exception {
   protected void shutDown() throws Exception {
     LOG.info("Shutting down Kaldb indexer.");
 
-    // Shutdown kafka consumer cleanly and then the chunkmanager so we can be sure, we have indexed
-    // the data we ingested.
+    // Shutdown kafka consumer
     try {
       kafkaConsumer.close();
     } catch (Exception e) {
       LOG.warn("Failed to close kafka consumer cleanly because of an exception.", e);
     }
 
-    chunkManager.stopAsync();
```

Comment:
Please add a comment explaining why we don't wait for chunk manager to be shutdown so we don't add it back later.

Reply:
Done.

Reply:
Missing comment?